### PR TITLE
Hive government

### LINKED ISSUE #1258: Allow Qiraji to reproduce
**Describe your suggestion in full detail below:**
Make qiraji characters fertile.

### LINKED ISSUE #1566: Hive government
**Describe your suggestion in full detail below:**
Adding a new government type for the aqir (the qiraji, nerubians, mantids).
Basic ideas:
- Allowed to hold castles and cities
- Allowed to revoke titles from non-hive rulers
- Ruler gets levy size bonus and dynasty opinion minus per every born child (immortal rulers are limited in children, so they get more bonus per child)
- Elective succession laws

### PR DESCRIPTION
**Pull request by zumbak04**
## Changelog:
- Added Hive government getting levy size bonus and dynasty opinion malus per child. The hivelords can sing to the swarm getting morale, opinion bonuses and build cost, tax maluses.
- The qiraji are allowed to reproduce.
- The nerubian religion uses scary rune CoAs.

# How to test:
There are 2 hivelords: the qiraji and nerubian ruler in 583's Icecrown.
I recommend testing the nerubian because he has more events (the qiraji lack many events because of the religion).
Try to find too weird events, decisions. Tell your opinion about balance.

Closes #1566, closes #1258.

### CODE REVIEW

File: localisation/00_zuwc_united_localisation.csv [hunk 1]
```diff
@@ -2882,19 +2882,19 @@ summon_gorak_tul_desc;With our hold over Drustvar secure, we are now able to sum
...
+GetPeasantType_peon;peon;;;;;;;;;;;;;x
+GetPeasantType_acolyte;acolyte;;;;;;;;;;;;;x
+GetPeasantsType_peasant;peasant;;;;;;;;;;;;;x
+GetPeasantsType_worker;worker;;;;;;;;;;;;;x
+GetCattlesType_pigs;pigs;;;;;;;;;;;;;x
+GetCattlesType_livestock;livestock;;;;;;;;;;;;;x
+song_of_swarm_cooldown;Recent Song of Swarm;;;;;;;;;;;;;x
+song_of_swarm_modifier;Song of Swarm;;;;;;;;;;;;;x
+song_of_swarm;Song of Swarm;;;;;;;;;;;;;x
+song_of_swarm_desc;You sing to the swarm, giving them purpose to fight, your words filling their minds. Your desires are their desires, and they don't hesitate to obey.;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:2897]:
```suggestion
song_of_swarm_desc;You sing to the swarm, giving them the desire to fight, your words filling their minds. Your will is their will, and they don't hesitate to obey.;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 2]
```diff
@@ -2882,19 +2882,19 @@ summon_gorak_tul_desc;With our hold over Drustvar secure, we are now able to sum
...
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+laid_eggs;Laid Eggs;;;;;;;;;;;;;x
+laid_eggs_desc;This character gets these bonuses and maluses per born child due to their government type.;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:2886]: I'm not too fond of the 'per born child' part of the sentence. My suggestion below could work, or you could replace it with 'per every child'. 
```suggestion
laid_eggs_desc;This character receives these bonuses and maluses per child due to their government type.;;;;;;;;;;;;;x
```

File: localisation/00_wc_government.csv [hunk 3]
```diff
@@ -19,3 +19,6 @@ dragon_government;Dragonflight;Vol draconique;Drachenschwarm;;;;;;;;;;;x
...
 society_government;Society;Soci�t�;Organisation;;;;;;;;;;;x
 society_government_desc;This society of individuals has progressed far beyond having common interests and motivations, and the membership of this society has grown to encompass entire populations of individuals. As such, this society has taken on more governmental reforms and leadership and is now recognized as a full, sovereign state contesting for resources like any other nation on Azeroth.;Les soci�t�s repr�sentent la forme politique des organisations.;Eine Regierungsform, die Gesellschaften und Organisationen widerspiegelt.;;;;;;;;;;;x
+hive_government;Hive;;;;;;;;;;;;;x
+hive_government_desc;An insectoid society built around kings and queens producing new members of the hive. Most larvae grow as submissive workers or soldiers, and only a few are born fertile and are able to reach the highest positions.;;;;;;;;;;;;x
+hive_government_FEATURES;�Y*�! Gets levy size bonus and dynasty opinion malus per every born child\n�Y*�! Can sing to the swarm every 50 years getting morale, opinion bonuses and build cost, tax maluses;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_wc_government.csv:24]: See below for my comments on the 'per born child' part. The second part had a few problems, so I tried to clean it up a little.
```suggestion
hive_government_FEATURES;�Y*�! Gets levy size bonus and dynasty opinion malus per child\n�Y*�! Can sing to the swarm every 50 years gaining morale and opinion bonuses, at the expense of higher building costs and tax maluses;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv (hunk 1 above)
Comment (zumbak04) [reply localisation/00_zuwc_united_localisation.csv]: Done in 21eff48138bfc52650c4c7c15c6ad1d6566bf162.

**Pull request merged**